tests/scripts/rebase.sh

```diff
@@ -2,7 +2,7 @@
 set -euxo pipefail
 
 # Rebase PRs on master to get latest changes
-if [[ $CI_COMMIT_REF_NAME == pr-* ]]; then
+if [[ $CI_COMMIT_REF_NAME == pr-* ]] && [[ $CI_COMMIT_REF_NAME != *-release-* ]]; then
   git config user.email "ci@kubespray.io"
   git config user.name "CI"
   git pull --rebase origin master
```

Comment:
You need to rebase on `release-2.13` instead of master for release branches. Ideally we should update that once the release branch is created or auto detect the target branch and rebase on it.

Reply:
Indeed I'll update, agreed about the auto detect didn't really find anything to do that but maybe that's worth researching a bit more